src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -703,10 +704,8 @@ export default class ResultsViewPage extends React.Component<
                                 {// we don't show the result tabs if we don't have valid query
                                 this.showTabs &&
                                     !this.resultsViewPageStore.genesInvalid &&
-                                    !this.resultsViewPageStore.isQueryInvalid &&
-                                    this.resultsViewPageStore
```

Comment:
I think @pvannierop added this for a reason.  We talked about moving it deeper but ...

Reply:
it means we load all mutation data before we show tabs

Reply:
why do we want to do that? the tabs handle their own loading state/indicators

Reply:
@pvannierop lets talk about whether we can put this further down or not.   but @adamabeshouse maybe we just need to add a loader when these conditions aren't true?

Reply:
@adamabeshouse i agree.  i know this came up before but i can't remember why it ended up here.